apps/avifenc.c

```diff
@@ -2305,6 +2305,7 @@ int main(int argc, char * argv[])
                     diag.error);
             goto cleanup;
         }
+        // avifCropRectRequiresUpsampling() does not need to be called for clap validation.
```

Comment:
Optional: This comment could be omitted. `avifCropRectRequiresUpsampling()` is a function intended for the decoder, so it seems irrelevant in avifenc.c.